pkg/selector/pod/selector.go

```diff
@@ -159,7 +158,7 @@ func selectSpecifiedPods(ctx context.Context, c client.Client, spec v1alpha1.Pod
 	for ns, names := range spec.Pods {
 		if !clusterScoped {
 			if targetNamespace != ns {
-				log.Info("skip namespace because ns is out of scope within namespace scoped mode", "namespace", ns)
+				log.L().WithName("pod-selector").Info("skip namespace because ns is out of scope within namespace scoped mode", "namespace", ns)
```

Comment:
It seems L is a deprecated func. Can we create a new logger?

Reply:
That's still a temporary solution. Because we have nowhere to store the logger.

Another solution is to change the function signature of `SelectAndFilterPods`, passing a logger from the caller, but changing the function signature seems a little much for this migration..


`log.L()` is designed to be used with this situation.